src/datepicker/datepicker.js

```diff
@@ -79,7 +69,7 @@ export default class Datepicker extends React.Component<
 
   formatDisplayValue(date: ?Date | Array<Date>) {
     const formatDisplayValue = this.props.formatDisplayValue || this.formatDate;
-    return formatDisplayValue(date, this.props.formatString);
+    return formatDisplayValue(date, this.props.formatString || 'YYYY/MM/dd');
```

Comment:
Is there a reason we moved it to here instead of default props?

Reply:
`formatString` is optional in types. I could have destructured with a default value, or in-lined like this